Resources/Textures/Objects/Weapons/Guns/Projectiles/projectiles2.rsi/meta.json

```diff
@@ -24,6 +24,24 @@
           0.05
         ]
       ]
+    },
+	{
+      "name": "rubber",
```

Comment:
Please don't mix tabs and spaces, if you use VS or Rider it should handle this for you.

Reply:
Please explain for i do not understand what you mean?

Reply:
On line 27, before the "}," there are four spacebar spaces.
On line 28, before the "{" there is one tab button space.
On line 29 you use spacebar spaces again.

Sloth would like it if you consistently use one of them. Visual Studio or Rider do this automatically.